tools/grub/Dockerfile

```diff
@@ -1,4 +1,6 @@
+# this is really hard to build. Do not change this version unless you must
```

Comment:
I appreciate the warning, but some context on why it's hard would be nice.

Reply:
I basically gave up on getting it to work with the newer versions. We are using a pretty old grub, no longer maintained, and will not get fixed. The warning was basically, "leave this alone, do not even try."

I looked about a year ago at replacing this with upstream grub. I think almost everything we customize already was upstreamed. Something was missing, but too much to remember now.

In any case, big red warning lights: "do not change this or suffer what you have brought upon yourself!"

Reply:
Similar issue. If we change a single thing in `tools/grub`, then we rebuild it and push it out, then change the downstream dependencies `tools/mkimage-iso-efi`, `tools/mkimage-qcow2-efi`, `tools/mkimage-raw-efi`, then rebuild those and push them out, then etc. etc.

We should avoid doing that at all costs here.